src/backend/gl/src/native.rs

```diff
@@ -15,13 +17,16 @@ pub type FrameBuffer = gl::types::GLuint;
 pub type Surface     = gl::types::GLuint;
 pub type Texture     = gl::types::GLuint;
 pub type Sampler     = gl::types::GLuint;
+pub(crate) type SetID     = gl::types::GLuint;
+pub(crate) type BindingID     = gl::types::GLuint;
```

Comment:
note that we already have `hal::pso::DescriptorSetIndex` and `hal::pso::DescriptorBinding`. Maybe we could use them here instead of defining new types?